src/test/scala/com/microsoft/hyperspace/index/rules/FilterIndexRuleTest.scala

```diff
@@ -125,6 +125,23 @@ class FilterIndexRuleTest extends HyperspaceRuleTestSuite {
     verifyTransformedPlanWithIndex(transformedPlan, indexName2)
   }
 
+  test("Verify FilterIndex rule is not applied for modified plan.") {
+    // Copied from test("Verify FilterIndex rule is applied when all columns are selected.")
```

Comment:
I think we can remove this comment. (it's hard to make it in sync if the test is updated, for example).